### PR TITLE
Promote provisionEmail and removeEmail actions on Teams channel to v1.0

### PR DESCRIPTION
**Pull request by shantanu1singh**
Promote provisionEmail and removeEmail actions on Teams channel to v1.0

### CODE REVIEW

File: api-reference/v1.0/api/channel-provisionemail.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+| :------------ | :------------------------ |
+| Authorization | Bearer {token}. Required. |
+
+## Request body
+
+Do not supply a request body for this method.
+
+## Response
+
+If the channel's email is provisioned successfully, this method returns a `200 OK` response code and a [provisionChannelEmailResult](../resources/provisionChannelEmailResult.md) object in the response body. The provisioned email address is in the **email** property.
```
[the remark this answers is not in the document]
Comment (shantanu1singh) [reply api-reference/v1.0/api/channel-provisionemail.md:46]: will update

File: api-reference/v1.0/api/channel-removeemail.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+## Response
+
+If successful, this action returns a `204 No Content` response code.
+
+## Example
+### Request
+
+<!-- {
+  "blockType": "request",
+  "sampleKeys": ["893075dd-2487-4122-925f-022c42e20265", "19:561fbdbbfca848a484f0a6f00ce9dbbd@thread.tacv2"],
```
[the remark this answers is not in the document]
Comment (shantanu1singh) [reply api-reference/v1.0/api/channel-removeemail.md:51]: Looks like I didn't add the sample HTTP request. Will fix

File: api-reference/v1.0/api/channel-provisionemail.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+doc_type: apiPageType
+---
+
+# channel: provisionEmail
+
+Namespace: microsoft.graph
+
+Provision an email address for a [channel](../resources/channel.md).
+
+Microsoft Teams does not automatically provision an email address for a channel by default. To have Teams provision an email address, you can call **provisionEmail**, or through the Teams user interface, select **Get email address**, which triggers Teams to generate an email address if it has not already provisioned one.
```
[the remark this answers is not in the document]
Comment (isvargasmsft) [reply api-reference/v1.0/api/channel-provisionemail.md:16]: @shantanu1singh the [Microsoft Writing Style](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36389) asks to [use contractions](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36389):
Microsoft Teams doesn't automatically...

File: api-reference/v1.0/api/channel-provisionemail.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+To remove a channel's provisioned email address, use the [removeEmail](channel-removeemail.md) method.
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/channel-provisionemail.md:18]: This phrase is not easy to understand for a global audience. Please rephrase to: 
* To remove the email address of a channel, use the ...

File: api-reference/v1.0/api/channel-removeemail.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+localization_priority: Normal
+ms.prod: "microsoft-teams"
+doc_type: apiPageType
+---
+
+# channel: removeEmail
+
+Namespace: microsoft.graph
+
+Remove the provisioned email address of a [channel](../resources/channel.md).
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/channel-removeemail.md:14]: Lines 3 and 14: Same observation above. This phrasing is not the best for a global audience. Remember this description is used by search engines to show results. If you need to remove the email address of a channel, how would you bing it? Please rephrase to:
Remove the email address of a channel.

Comment (isvargasmsft) [reply api-reference/v1.0/api/channel-removeemail.md:14]: If the beta topic has the same description, would you mind updating it too?

File: api-reference/v1.0/resources/channel.md [hunk 6]
```diff
@@ -40,7 +40,8 @@ where files are shared, and where tabs are added.
 |[Get tab in channel](../api/channel-get-tabs.md) | [teamsTab](teamstab.md) | Get a specific tab pinned to a channel.|
 |[Update tab in channel](../api/channel-patch-tabs.md) | [teamsTab](teamstab.md) | Updates the properties of a tab in a channel.|
 |[Remove tab from channel](../api/channel-delete-tabs.md) | None | Remove (unpin) a tab from a channel.|
-
+|[Provision channel email address](../api/channel-provisionemail.md) |[provisionChannelEmailResult](../resources/provisionchannelemailresult.md)| Provision email address.|
```
[the remark this answers is not in the document]
Comment (isvargasmsft) [reply api-reference/v1.0/resources/channel.md:43]: As pe our conversation on the side: Provision an email address for the channel.

File: api-reference/v1.0/resources/channel.md [hunk 7]
```diff
@@ -40,7 +40,8 @@ where files are shared, and where tabs are added.
...
+|[Remove channel email address](../api/channel-removeemail.md) | None | Remove email address.|
```
[the remark this answers is not in the document]
Comment (isvargasmsft) [reply api-reference/v1.0/resources/channel.md:44]: As per our conversation on the side: 
Remove the email address of the channel.

File: api-reference/v1.0/api/channel-provisionemail.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (isvargasmsft) [reply api-reference/v1.0/api/channel-provisionemail.md:16]: Please also bold the word **channel**.

File: api-reference/v1.0/api/channel-provisionemail.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+| :------------ | :------------------------ |
+| Authorization | Bearer {token}. Required. |
+
+## Request body
+
+Do not supply a request body for this method.
+
+## Response
+
+If the channel's email is provisioned successfully, this method returns a `200 OK` response code and a [provisionChannelEmailResult](../resources/provisionChannelEmailResult.md) object in the response body. The provisioned email address is in the `email` property.
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/channel-provisionemail.md:46]: Let's rephrase for simplicity: If successful, this method returns...

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Thank you for your work, Shantanu. 

Please review some rephrasing suggestions and let me know if you have any questions.

Here to help!

File: api-reference/v1.0/resources/channel.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (shantanu1singh) [reply api-reference/v1.0/resources/channel.md:43]: done

File: api-reference/v1.0/resources/channel.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (shantanu1singh) [reply api-reference/v1.0/resources/channel.md:44]: done

File: api-reference/v1.0/api/channel-provisionemail.md (hunk 4 above)
Comment (shantanu1singh) [reply api-reference/v1.0/api/channel-provisionemail.md:18]: done

File: api-reference/v1.0/api/channel-removeemail.md (hunk 5 above)
Comment (shantanu1singh) [reply api-reference/v1.0/api/channel-removeemail.md:14]: sure, done

File: api-reference/v1.0/api/channel-provisionemail.md (hunk 8 above)
Comment (shantanu1singh) [reply api-reference/v1.0/api/channel-provisionemail.md:46]: sure, done

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[2e6011d](https://github.com/shantanu1singh/microsoft-graph-docs/commits/2e6011de1226481c53f1d80d4b1e853f89dea574)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/channel-provisionemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/beta/api/channel-provisionemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-provisionemail?view=graph-rest-beta&branch=pr-en-us-13982) (graph-rest-beta) |
[api-reference/beta/api/channel-removeemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/beta/api/channel-removeemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-removeemail?view=graph-rest-beta&branch=pr-en-us-13982) (graph-rest-beta) |
[api-reference/beta/resources/channel.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/beta/resources/channel.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-beta&branch=pr-en-us-13982) (graph-rest-beta) |
[api-reference/v1.0/api/channel-provisionemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/api/channel-provisionemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-provisionemail?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/api/channel-removeemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/api/channel-removeemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-removeemail?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/channel.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/resources/channel.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/provisionchannelemailresult.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/resources/provisionchannelemailresult.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/provisionchannelemailresult?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/toc.yml](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-provisionemail?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[changelog/Microsoft.Teams.GraphSvc.json](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/changelog/Microsoft.Teams.GraphSvc.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109011635205462-13982%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=Dl9ir%2BVKXzt76WACpDda76kvj%2BUJeJ08cj0smwh82%2FY%3D&st=2021-09-01T16%3A34%3A03Z&se=2021-10-02T16%3A39%3A03Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by isvargasmsft — APPROVED** [review]

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[7cfd491](https://github.com/shantanu1singh/microsoft-graph-docs/commits/7cfd491be6951cbe2c9229d4102fda883232e6ca)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/channel-provisionemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/beta/api/channel-provisionemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-provisionemail?view=graph-rest-beta&branch=pr-en-us-13982) (graph-rest-beta) |
[api-reference/beta/api/channel-removeemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/beta/api/channel-removeemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-removeemail?view=graph-rest-beta&branch=pr-en-us-13982) (graph-rest-beta) |
[api-reference/beta/resources/channel.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/beta/resources/channel.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-beta&branch=pr-en-us-13982) (graph-rest-beta) |
[api-reference/v1.0/api/channel-provisionemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/api/channel-provisionemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-provisionemail?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/api/channel-removeemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/api/channel-removeemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-removeemail?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/channel.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/resources/channel.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/provisionchannelemailresult.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/resources/provisionchannelemailresult.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/provisionchannelemailresult?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/toc.yml](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-provisionemail?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[changelog/Microsoft.Teams.GraphSvc.json](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/changelog/Microsoft.Teams.GraphSvc.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109021332397984-13982%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=nIl2p9Nv1QmUTlYbTl0YyCLJdVZBSrnejTNGcQwdkyQ%3D&st=2021-09-02T13%3A30%3A24Z&se=2021-10-03T13%3A35%3A24Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[47a3a33](https://github.com/shantanu1singh/microsoft-graph-docs/commits/47a3a33a3ae525e2c60a7629d05f3bbad7944bca)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/channel-provisionemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/beta/api/channel-provisionemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-provisionemail?view=graph-rest-beta&branch=pr-en-us-13982) (graph-rest-beta) |
[api-reference/beta/api/channel-removeemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/beta/api/channel-removeemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-removeemail?view=graph-rest-beta&branch=pr-en-us-13982) (graph-rest-beta) |
[api-reference/beta/resources/channel.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/beta/resources/channel.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-beta&branch=pr-en-us-13982) (graph-rest-beta) |
[api-reference/v1.0/api/channel-provisionemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/api/channel-provisionemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-provisionemail?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/api/channel-removeemail.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/api/channel-removeemail.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-removeemail?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/channel.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/resources/channel.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/provisionchannelemailresult.md](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/resources/provisionchannelemailresult.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/provisionchannelemailresult?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[api-reference/v1.0/toc.yml](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/api-reference/v1.0/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/channel-provisionemail?view=graph-rest-1.0&branch=pr-en-us-13982) (>= graph-rest-1.0) |
[changelog/Microsoft.Teams.GraphSvc.json](https://github.com/shantanu1singh/microsoft-graph-docs/blob/shsi/channelemailv1/changelog/Microsoft.Teams.GraphSvc.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109021343151753-13982%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=QAsINYvxcZLGG3JvrDfSLtRSg5UFP6CwxfJgHugGsxc%3D&st=2021-09-02T13%3A39%3A45Z&se=2021-10-03T13%3A44%3A45Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**